lib/propolis/src/hw/ps2ctrl.rs

```diff
@@ -417,9 +660,10 @@ const PS2K_R_SELF_TEST_PASS: u8 = 0xaa;
 
 const PS2K_TYPEMATIC_MASK: u8 = 0x7f;
 
+// XXX(JPH): Is there a reason this needs to be this size?
```

Comment:
I *think* I just picked an arbitrary for this when I scraped together the initial PS/2 emulation